### PR TITLE
External identities page added

### LINKED ISSUE #239: User detail - external identities
* There should be added a page to the user detail with his external identities. 

* It should work similarly to the old GUI.

### PR DESCRIPTION
**Pull request by xkostka2**
* External identities page has been added to user detail
* user-ext-sources-list and remove-user-ext-source-dialog has been moved to shared library
* Dialog for adding user ext sources has been added
* Resolves #239

### CODE REVIEW

File: apps/admin-gui/src/app/shared/components/user-detail-page/user-identities/user-identities.component.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  refreshTable(){
+    this.loading = true;
+    this.usersManagerService.getRichUserExtSources(this.userId).subscribe(userExtSources =>{
+      this.userExtSources = userExtSources;
+      this.loading = false;
+    })
+  }
+
+  addIdentity() {
+    const dialogRef = this.dialog.open(AddUserExtSourceDialogComponent, {
```
Comment (DanoFecko) [inline apps/admin-gui/src/app/shared/components/user-detail-page/user-identities/user-identities.component.ts:50]: Adam recently added default config for dialogs (https://github.com/CESNET/perun-web-apps/pull/247), there is method in perun-utils to get it (getDefaultDialogConfig)

**Comment by DanoFecko** [comment]
There is error when opening Add dialog: Error 172bd4964af: Error 172bd4964af: Policy with name getExtSources_policydoes not exists in the PerunPoliciesContainer.

**Comment by DanoFecko** [comment]
Add dialog looks weird. Try to unify it with other add dialogs.

File: libs/perun/components/src/lib/remove-user-ext-source-dialog/remove-user-ext-source-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <tr *matRowDef="let richUserExtSource; columns: displayedColumns;" mat-row></tr>
+      </table>
+    </div>
+
+    <div mat-dialog-actions>
+
+      <button
+        (click)="onCancel()"
+        class="ml-auto"
+        mat-stroked-button>
```
Comment (xkureck) [inline libs/perun/components/src/lib/remove-user-ext-source-dialog/remove-user-ext-source-dialog.component.html:29]: please use mat-flat-button as in all other cancel buttons we have in our app

File: apps/admin-gui/src/app/shared/components/dialogs/add-user-ext-source-dialog/add-user-ext-source-dialog.component.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  selector: 'app-add-user-ext-source-dialog',
+  templateUrl: './add-user-ext-source-dialog.component.html',
+  styleUrls: ['./add-user-ext-source-dialog.component.scss']
+})
+export class AddUserExtSourceDialogComponent implements OnInit {
+
+  extSources: ExtSource[] = [];
+  filteredExtSources: Observable<ExtSource[]>;
+  loas = [0,1,2,3]
+  loa = 0;
```
Comment (xkureck) [inline apps/admin-gui/src/app/shared/components/dialogs/add-user-ext-source-dialog/add-user-ext-source-dialog.component.ts:28]: ;

File: apps/admin-gui/src/app/shared/components/dialogs/add-user-ext-source-dialog/add-user-ext-source-dialog.component.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+  onCancel(){
+    this.dialogRef.close(false);
+  }
+
+  onAdd(){
+    this.loading = true;
+    const ues: UserExtSource = {
+      beanName: '', extSource: this.extSourcesControl.value, id: 0, login: this.loginControl.value, loa: this.loa, userId: this.data.userId
+    }
```
Comment (xkureck) [inline apps/admin-gui/src/app/shared/components/dialogs/add-user-ext-source-dialog/add-user-ext-source-dialog.component.ts:74]: ;

**Comment by xkureck** [comment]
Also rebase

**Pull request merged**